src/legacy/core_plugins/kibana/public/visualize/listing/visualize_listing_table.js

```diff
@@ -103,10 +103,31 @@ class VisualizeListingTableUi extends Component {
         render: (field, record) =>  (
           <span>
             {this.renderItemTypeIcon(record)}
-            {record.type.title}
+            {record.typeTitle}
             {this.getExperimentalBadge(record)}
           </span>
         )
+      },
+      {
+        field: 'canEdit',
```

Comment:
The old "edit" link was gray as long as not hovered and included a tooltip - we should make sure to make this look identical:

Before:
<img width="157" alt="Screenshot 2019-08-15 at 14 07 47" src="https://user-images.githubusercontent.com/1508364/63093595-2d5ab700-bf66-11e9-8631-f878b105234c.png">

After:
<img width="146" alt="Screenshot 2019-08-15 at 14 08 43" src="https://user-images.githubusercontent.com/1508364/63093617-44010e00-bf66-11e9-9397-60797a8dccbc.png">

Reply:
Hm. Righto.